src/main/webapp/index.jsp

```diff
@@ -17,32 +17,157 @@
-      </ul>
+      </div>
+      </div> <!-- End Active-->
+      <div class="item">
+        <img src="https://u.imageresize.org/v2/e5a1c204-c371-4bdf-ac5c-8b30a1c9dbce.jpeg" style="width:100%;">
+      </div>
+    </div>
+    <!-- Start Slider Controls -->
+    <a class="left carousel-control" href="#myCarousel" role="button" data-slide="prev">
+      <span class="glyphicon glyphicon-chevron-left" aria-hidden="true"></span>
+      <span class="sr-only">Previous</span>
+    </a>
+    <a class="right carousel-control" href="#myCarousel" role="button" data-slide="next">
+      <span class="glyphicon glyphicon-chevron-right" aria-hidden="true"></span>
+      <span class="sr-only">Next</span>
+    </a>
+  </div>
+  <!-- Ends Slider -->
+
+  <div class="container text-center">
+    <h1>Meet the CodeByters!</h1>
+  </div>
+  <div class="container">
+    <!-- Jaiyi Wang -->
+    <div class"row">
+      <div class="col-md-6">
+        <br>
+        <br>
+        <strong><h3>Jaiyi Wang - Virginia Tech</h3></strong>
+        <p>
+          Jaiyi is a driven, third year, Computer Science student at Virginia Tech in Blacksburg, VA!
+        </p>
+      </div>
+      <div class="col-md-6">
+        <br>
+        <img src="http://i1067.photobucket.com/albums/u437/donnedebnam1/imageedit_12_8988254368_zpssfyxzbsx.gif" style="height:300px;" class="img-responsive">
+          <br>
+      </div>
+    </div>
+
+    <!-- Donnell Debnam -->
+    <div class"row">
+      <div class="col-md-6">
+        <br>
+        <strong><h3>Donnell Debnam Jr - Hampton University</h3></strong>
+        <p>
+          Donnell is a second year, Computer Science/Cybersecurity student currently attending Hampton University.
+          He is a highly motivated and resilient student, experienced in both web and mobile development, and has a keen interest in Arduino Robotics, AI/NLP, and forensics.
+        </p>
+      </div>
+      <div class="col-md-6">
+
+        <img src="http://i1067.photobucket.com/albums/u437/donnedebnam1/mypic1_zps8po8o6ss.gif" style="height:300px;" class="img-responsive">
+        <br>
+      </div>
+    </div>
+
+    <!-- Saroj Bhatta-->
+    <div class"row">
+      <div class="col-md-6">
+        <br>
+        <strong><h3>Saroj Bhatta - Claflin University</h3></strong>
+        <p>
+          Saroj is a bright, second year, Computer Science student at Claflin University in Orangeburg, SC.
+        </p>
+      </div>
+      <div class="col-md-6">
+        <img src="http://i1067.photobucket.com/albums/u437/donnedebnam1/imageedit_7_2530905731_zpshmm3ddnp.gif" style="height:300px;" class="img-responsive">
+        <br>
+      </div>
     </div>
+
+    <!-- Anthony Ngonma-->
+    <div class"row">
+      <div class="col-md-6">
+        <br>
+        <strong><h3>Anthony Ngoma - Cornell University</h3></strong>
+        <p>
+          Anthony is a second year Computer Science student currently attending the illustrious Cornell University located in Ithica, NY.
```

Comment:
"Anthony is a second year CS/CE student attending the illustrious Cornell University located in Ithaca, NY. In his free time, Anthony enjoys playing soccer(a lot!) and learning new things such as cooking."

Reply:
Got it!